Tools/PRAccelerator/PRAcceleratorPipeline.yml

```diff
@@ -0,0 +1,98 @@
+trigger: none
+
+stages:
+- stage: __default
+  jobs:
+  - job: Job_1
+    displayName: Agent job 1
+    pool:
+      name: local
+    steps:
+    - checkout: self
+      persistCredentials: True
+    - task: PowerShell@2
+      displayName: Print variables values
+      inputs:
+        targetType: inline
+        script: >+
+          Write-Host "--------------------------------------------------------------------------"
+
+          Write-Host "Your variable values are:" -ForegroundColor Yellow
+
+          Write-Host "BranchName: $(BranchName)"
+
+          Write-Host "BuildTools.EnvironmentUrl: $(BuildTools.EnvironmentUrl)"
+
+          Write-Host "SolutionName: $(SolutionName)"
+
+          Write-Host "--------------------------------------------------------------------------"
+
+          Write-Host "Example variable values are:" -ForegroundColor Yellow
+
+          Write-Host "BranchName: users/my-alias/task-123456"
+
+          Write-Host "BuildTools.EnvironmentUrl: https://scshubplatdev.crm.dynamics.com/"
```

Comment:
please update org name to a fictitious value

Reply:
Done